docs/CaliptraHardwareSpecification.md

```diff
@@ -457,9 +469,9 @@ When the RV core wakes up, all clocks are enabled. However, when the core is hal
 
 * JTAG accesses
 
-* APB transactions
+* AXI transactions
 
-Activity on the APB interface only wakes up the SoC IFC clock. All other clocks remain off until any other condition is met or the core exits the halt state.
+Activity on the AXI interface only wakes up the SoC IFC clock. All other clocks remain off until any other condition is met or the core exits the halt state.
 
 | Cpu_halt_status | PSEL | Generic input wires <br>\|\| fatal error <br>\|\| debug/scan mode  <br> \|\|JTAG access | Expected behavior |
```

Comment:
Access table no longer uses "PSEL" - it's a combination of arvalid and awvalid and "s_axi_active", which tracks whether requests are outstanding or not.

Reply:
what should we call it?